components/dropdown/dropdown-content-styles.js

```diff
@@ -13,50 +13,47 @@ export const dropdownContentStyles = css`
 		z-index: 1000; /* position on top of floating buttons */
 	}
 
-	:host([opened]) {
-		display: inline-block;
-	}
-
 	:host([opened]) {
 		-webkit-animation: d2l-dropdown-animation 300ms ease;
 		animation: d2l-dropdown-animation 300ms ease;
+		display: inline-block;
 	}
 
 	:host([opened-above]) {
-		bottom: calc(100% + var(--d2l-dropdown-verticaloffset, 20px));
-		top: auto;
 		-webkit-animation: d2l-dropdown-above-animation 300ms ease;
 		animation: d2l-dropdown-above-animation 300ms ease;
+		bottom: calc(100% + var(--d2l-dropdown-verticaloffset, 20px));
+		top: auto;
 	}
 
 	.d2l-dropdown-content-pointer {
-		position: absolute;
-		display: inline-block;
 		clip: rect(-5px, 21px, 8px, -7px);
-		top: -7px;
+		display: inline-block;
 		left: calc(50% - 7px);
+		position: absolute;
+		top: -7px;
 		z-index: 1;
 	}
 
 	.d2l-dropdown-content-pointer > div {
 		background-color: #ffffff;
 		border: 1px solid var(--d2l-color-mica);
 		border-radius: 0.1rem;
-		box-shadow: -4px -4px 12px -5px rgba(73, 76, 78, .2); /* ferrite */
+		box-shadow: -4px -4px 12px -5px rgba(73, 76, 78, 0.2); /* ferrite */
 		height: 16px;
-		width: 16px;
-		transform: rotate(45deg);
 		-webkit-transform: rotate(45deg);
```

Comment:
Note for devs: ignore browser prefixes when ordering.

Reply:
Except they should precede the non-prefixed version so that browsers who understand both use the standard one.

Reply:
I'll see if I can find a way to enforce this.

Reply:
As it turns out, the order plugin already enforces this.  :)